File: ObjectiveCTutorial/ObjectiveCTutorial/ViewController.m
//
//  ViewController.m
//  ObjectiveCTutorial
//
//  Created by mcs on 10/24/19.
//  Copyright © 2019 EricOusouphan. All rights reserved.
//

#import "ViewController.h"

@interface ViewController ()

@end

@implementation ViewController

- (void)viewDidLoad {
  [super viewDidLoad];
  
  [[[NSURLSession.sharedSession dataTaskWithURL:NSURL alloc]
    initwithString: @"www.google.com"] completionHandler:^(NSData * _Nullable data, NSURLResponse *_Nullable response, NSError *_Nullable error){
    id jsonObject = [NSJSONSerialization JSONObjectWithData:data options:kNilOptions error: (NSError *__autoreleasing)];
    if (data) {
      
    }
  }]
  
  
}


@end

File: CocoaPodProject/CocoaPodProject/ViewController.swift
//
//  ViewController.swift
//  CocoaPodProject
//
//  Created by mcs on 10/31/19.
//  Copyright © 2019 EricOusouphan. All rights reserved.
//

import UIKit
import Alamofire
class ViewController: UIViewController {

  override func viewDidLoad() {
    super.viewDidLoad()
    request("https://www.google.com", method: .get, parameters: nil, encoding: JSONEncoding(), headers: nil).responseData { (response) in
      print(response.data)
      print(response.error)
      print.(response.response)
    }
  }


}

